config/routes.rb

```diff
@@ -1,3 +1,21 @@
 Rails.application.routes.draw do
+
+  get 'auth/:provider/callback', to: 'sessions#create'
+  get 'auth/failure', to: redirect('/')
+  get 'signout', to: 'sessions#destroy', as: 'signout'
+
+  resources :sessions, only: [:create, :destroy]
+  resources :home, only: [:show]
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: "home#show"
+
+  # resources :businesses, only: [:index, :show]
+  get '/businesses', to: 'businesses#index'
+  get '/businesses/:id', to: 'businesses#show'
+  resources :cities, only: [:index, :show]
```

Comment:
This might just be a personal preference, but if you're only doing index/show, I would not use resources and then restrict it, I would probably write it out by hand. If you wanted ALL but two routes, I would use this syntax and instead of `only, `except`.